Dockerfile-php8

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+
+RUN apt-get update && apt-get install -y \
+    apt-transport-https \
+    curl \
+    git \
+    locales \
+    mysql-client \
+    nginx \
+    software-properties-common \
+    unzip \
+    tzdata \
+    vim
+
+# System-level PHP setup
+RUN apt-get install -y \
+    php8.1 php8.1-curl php8.1-fpm php8.1-gd php8.1-mbstring php8.1-mysql php8.1-xml
+RUN mkdir /run/php # For the php-fpm socket file
```

Comment:
`/var/run/php/` is used in the nginx config below...

And does this actually need doing manually?

Reply:
I don't remember the details - I did need it at some point.

Reply:
It appears that `/var/run` is a symlink to `/run` on Ubuntu.

```
root@af5f1b02b4f1:/# ls -lah /var/run
lrwxrwxrwx 1 root root 4 Nov 30  2022 /var/run -> /run
```

Still need to double-check if this needs to be done manually.

Reply:
> `/var/run/php/` is used in the nginx config below...
> 
> And does this actually need doing manually?

OK, it's because we run `php-fpm` directly instead of going through `systemd`...